### ISSUE TITLE
Negative drift in sequence

### ISSUE THREAD

**Issue by coldfix** [coldfix/madx-dummy#93]
**Originally by _luke.thompson_ on 2012-01-27 18:26:29 +0100**

___
date: 2011.10.28
I'm having a problem with MAD-X 4.01.12; I've also reproduced this in MAD-X 5.00.08 which I believe is the latest version. When using SAVE to output a sequence file, MAD-X sometimes has problems with reading in the file it's just generated; it claims there are negative drifts, usually of a small value, for instance 1e-6. I've attached some files which, for me, produce this error. It appears truncation of position values is occurring in the saved sequence. Is there any way around this?

**Comment by coldfix** [comment]
**Originally by _ldeniau_ on 2012-01-27 18:27:12 +0100**

___
**Attachment added:** [`seq.test.madx`](https://raw.githubusercontent.com/coldfix/madx-tickets/master/attachments/93/seq.test.madx) (0.2 KiB)

**Comment by coldfix** [comment]
**Originally by _ldeniau_ on 2012-01-27 18:28:03 +0100**

___
**Attachment added:** [`v1.0.seq.zip`](https://raw.githubusercontent.com/coldfix/madx-tickets/master/attachments/93/v1.0.seq.zip) (72.1 KiB)

**Comment by coldfix** [comment]
**Originally by _ldeniau_ on 2012-03-12 18:08:31 +0100**

___
date: 2012.03.12, from L. Thompson
I don't think this is a defect, rather a misunderstanding. I presumed that MADX would output values to the precision it was given them, so on reading in a sequence file with high precision values for elements I thought it would output the same values to a sequence file. I fixed my problem by using the SET, FORMAT="" command.
